WordPress/Classes/Services/PostService.m

```diff
@@ -586,7 +586,10 @@ - (void)updatePost:(AbstractPost *)post withRemotePost:(RemotePost *)remotePost
     post.mt_excerpt = remotePost.excerpt;
     post.wp_slug = remotePost.slug;
     post.suggested_slug = remotePost.suggestedSlug;
-    post.revisions = [remotePost.revisions copy];
+    
+    if ([remotePost.revisions wp_isValidObject]) {
```

Comment:
`wp_isValidObject` stands out as a weird name. 

At some point we should drop the `wp_` 😬

Reply:
I'd like to remove the whole category at some point!